search/lib/api/wfs.js

```diff
@@ -37,7 +38,13 @@ async function getCollections (request, response) {
     const { currPage, prevResultsLink, nextResultsLink } = generateNavLinks(event);
 
     const provider = request.params.providerId;
-    const collections = await cmr.findCollections({ provider_short_name: provider });
+    const params = Object.assign(
+      { provider_short_name: provider },
+      //Used for pagination
+      //await cmr.convertParams(provider, request.query)
+      { page_num: request.query.page}
```

Comment:
Ok, as discussed before, yes makes sense to allow pagination here (which I should not have removed before), but as indicated in earlier comment about providers, if `page` parameter is accepted, we should probably also accept `limit` (ie page size)

Reply:
We will use cmr.convertParams as mentioned above because user can pass in all the parameters allowed, not limited to page and limit.